front/app/containers/IdeasShow/components/IdeaTitle.tsx

```diff
@@ -31,15 +31,9 @@ interface Props {
   idea: IIdea;
   projectId: string;
   translateButtonClicked: boolean;
-  showActions: boolean;
```

Comment:
This only showed on phones. Now it always shows because I took out IdeaMoreActions from DesktopTopBar in order to move DesktopTopBar out of the main landmark. See files below